LEDProjector/v1/_init.yaml

```diff
@@ -0,0 +1,9 @@
+device_type: T4
+connection_type: TCP
+identifier: 140.252.147.123
```

Comment:
This seems like it belongs in a "specialized" configuration. 

I recommend setting the default to "127.0.0.1" and then create a separate `_tucson.yaml` config file with only this entry. Also, it is better to use the host name, e.g.;

```
identifier: ledprojectorlj.tu.lsst.org
```

Reply:
I agree, let's make a `_tucson.yaml` with this IP address as it will change when we move to the summit. Otherwise looks good